GSA/digital-gov-drupal
# DIGITAL-43: Vocabulary for glossary terms

## Jira ticket

[DIGITAL-43](https://cm-jira.usa.gov/browse/DIGITAL-43)

## Relates to 

[Digital-60](https://github.com/GSA/digital-gov-drupal/pull/78)
[Digital-61](https://github.com/GSA/digital-gov-drupal/pull/80 )

)
## Purpose

- [x] A new vocabulary (HCD Glossary) is created with fields of title & body or description

### QA/Testing instructions

1. Pull branch
2. Drush CIM
3. lando rebuild && lando si
4. Taxonomy module should be enabled.

1. Check that the HCD Glossary exists and that all terms are listed in glossary.

## Checklist for the Developer


## Checklist for the Peer Reviewers
